app/components/decisions/DecisionTranslation.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+    <div>
+        <div>
+            <table>
+                <thead>
+                    <tr>
+                        <td>English</td>
+                        <td>
+                            <select class="form-control" id="languages" v-model="selectedLanguage">
+                                <option 
+                                    v-for="lang in Object.keys(languages)"
+                                    :key="lang" 
+                                    :value="lang">
+                                    {{languages[lang]}}
+                                </option>
+                            </select>
+                        </td>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr v-for="(row, index) in docRows" :key="index">
+                        <td><span v-html="row.en" /></td>
+                        <td>
+                            <div v-if="docRows[index].editor">
+                                <ckeditor v-model="editorHtml" :editor="editorType" :config="editorConfig"></ckeditor>
+                                <div class="float-right">
+                                    <span class="btn text-success" @click="save(row, index)"><i class="fa fa-check"></i></span>
+                                    <span class="btn text-danger" @click="cancel(row, index)"><i class="fa fa-times"></i></span>
+                                </div>
+                            </div>
+                            <span v-else v-html="row[selectedLanguage]"  @click="handleRowClick(row, index)" />
+                        </td>
+                    </tr>
+                </tbody>
+            </table>
+        </div>
+    </div>
+</template>
+
+<script>
+import { cloneDeep } from 'lodash'
+import ClassicEditor from 'ckeditor5';
+import { component as ckeditor } from 'vue-ckeditor5'
+import docRows from '~/data/decisions/text-info.json';
+import languages from '~/components/languages.js';
+
+export default {
+    name: 'DecisionEditTranslations',
+    components : { ckeditor },
+    data() {
+        return {
+            editorType: ClassicEditor,
+            editorConfig: {},
+            editorHtml: '',
+            selectedLanguage: 'fr',
+            docRows,
+        }
+    },
+    computed: {
+        languages() {
+            const langs = cloneDeep(languages);
+            delete langs.en;
+            return langs;
+        } 
+    },
+    methods: {
+        handleRowClick,
+        save, 
+        cancel
+    }
+}
+
+function handleRowClick(row, index) {
+    row.editor = true;
+    this.editorHtml = this.docRows[index][this.selectedLanguage] || '';
+    this.$set(this.docRows, index, row);
+}
```

Comment:
rename to `edit`
the editedHtml must be per-row because you can have multiple edit at same time. 

```suggestion
function edit(row) {
    row.editor = true;
    row.editorHtml = row[this.selectedLanguage] || '';
}
```

Reply:
Done!